lib/src/DatePicker/Calendar.jsx

```diff
@@ -85,7 +87,10 @@ export class Calendar extends Component {
   };
 
   handleChangeMonth = (newMonth) => {
-    this.setState({ currentMonth: newMonth });
+    this.setState(
+      { currentMonth: newMonth },
+      () => this.props.onSelectMonth(this.state.currentMonth),
```

Comment:
If `onSelectMonth` will be undefined there will be an error cannot call undefined as function

Reply:
That's why I defined a default value :)